xcell/mappers/mapper_CatWISE.py

```diff
@@ -37,17 +40,39 @@ def get_catalog(self):
                  self.config.get('flux_max_W1', 16.4))]
         return self.cat_data
 
+    # Correction to Density
+    def get_ecliptic_correction(self):
+        pixarea_deg2 = (hp.nside2resol(self.nside, arcmin=True)/60)**2
+        # Transforms equatorial to ecliptic coordinates
+        r = hp.Rotator(coord=['C', 'E'])
+        # Get equatorial coordinates
+        theta_EQ, phi_EQ = hp.pix2ang(self.nside, np.arange(self.npix))
+        # Rotate to ecliptic
+        theta_EC, phi_EC = r(theta_EQ, phi_EQ)
+        # Make a map of ecliptic latitude
+        ec_lat_map = 90-np.degrees(theta_EC)
+        # this hard-coded number stems from the fit in 2009.14826
+        correction = 0.0513 * np.abs(ec_lat_map) * pixarea_deg2
+        return correction
+
     # Density Map
-    def get_signal_map(self, apply_galactic_correction=True):
+    def get_signal_map(self):
         if self.delta_map is None:
             d = np.zeros(self.npix)
             self.cat_data = self.get_catalog()
             self.mask = self.get_mask()
+            # ecliptic latitude correction -- SvH 5/3/22
+            if self.apply_ecliptic_correction:
+                correction = self.get_ecliptic_correction()
+            else:
+                correction = np.zeros_like(d)
```

Comment:
For readability I would move this step after calculating the `nmap` ; i.e. just before line 72.
Also `correction = self.get_ecliptic_correction()  --> correction = self._get_ecliptic_correction()`